# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/fields/field_types.js
# Commit: 1b8c42577e24
# Full Hash: 1b8c42577e2410dbbe050bb30b0ccced9d7c3771
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-20 10:52:37
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r 28370bf14b97 -r 1b8c42577e24 js/src/tests/non262/fields/field_types.js
--- a/js/src/tests/non262/fields/field_types.js	Tue Mar 19 22:02:49 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,25 +0,0 @@
-// * * * THIS TEST IS DISABLED - Fields are not fully implemented yet
-
-class C {
-    [Math.sqrt(4)];
-    [Math.sqrt(8)] = 5 + 2;
-    "hi";
-    "bye" = {};
-    2 = 2;
-    0x101 = 2;
-    0o101 = 2;
-    0b101 = 2;
-    NaN = 0; // actually the field called "NaN", not the number
-    Infinity = 50; // actually the field called "Infinity", not the number
-    // all the keywords below are proper fields (?!?)
-    with = 0;
-    //static = 0; // doesn't work yet
-    async = 0;
-    get = 0;
-    set = 0;
-    export = 0;
-    function = 0;
-}
-
-if (typeof reportCompare === "function")
-  reportCompare(true, true);